multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -682,16 +683,33 @@ def fit(
             return self
 
         if self._problem_type == OBJECT_DETECTION:
-            self._detection_anno_train = train_data
-            train_data = from_coco_or_voc(train_data, "train")
-            if tuning_data is not None:
-                self.detection_anno_train = tuning_data
-                tuning_data = from_coco_or_voc(tuning_data, "val")
-                if max_num_tuning_data is not None:
-                    if len(tuning_data) > max_num_tuning_data:
-                        tuning_data = tuning_data.sample(
-                            n=max_num_tuning_data, replace=False, random_state=seed
-                        ).reset_index(drop=True)
+            if isinstance(train_data, str):
+                self._detection_anno_train = train_data
+                train_data = from_coco_or_voc(train_data, "train")  # TODO: Refactor to use convert_data_to_df
```

Comment:
Need an example or tutorial to visualize what the dataframe looks like, because one image path can have multiple bboxes with coordinates. Users with no experience in object detection may not know what the format looks like. In addition, because we support dataframe now, users can directly prepare data in dataframe format without converting to coco/voc format first.

Reply:
Yea totally agree. I think it is necessary to do such a tutorial for `.fit()` with df to make sure they are passing the right format.